SpatialGDK/Source/SpatialGDK/Private/Utils/SchemaDatabase.cpp

```diff
@@ -0,0 +1,6 @@
+#include "Utils/SchemaDatabase.h"
+
+USchemaDatabase::USchemaDatabase() : NextAvailableComponentId(SpatialConstants::STARTING_GENERATED_COMPONENT_ID)
```

Comment:
If this is all that's here, you could just default initialize the variable inline.

Reply:
deleted the SchemaDatabase.cpp file